go/sqldb/conn.go

```diff
@@ -18,12 +18,11 @@ import (
 type NewConnFunc func(params ConnParams) (Conn, error)
 
 // conns stores all supported db connection.
-var conns = make(map[string]NewConnFunc)
-
-var mu sync.Mutex
-
-// DefaultDB decides the default db connection.
-var DefaultDB string
+var (
+	mu          sync.Mutex
+	conns       = make(map[string]NewConnFunc)
+	defaultConn NewConnFunc
```

Comment:
I'd move this one up before the mutex, then add a comment before the mutex to say that it protects the map.